design-principles.md

```diff
@@ -92,10 +163,18 @@ API design
 Bugs
 ----
 
-"Don't lick the cookie": Only assign a bug to yourself when you are
-actively working on it. If you're not working on it, leave it
-unassigned. Don't assign bugs to people unless you know they are going
-to work on it.
+Only assign a bug to yourself when you are actively working on it. If
+you're not working on it, leave it unassigned. Don't assign bugs to
+people unless you know they are going to work on it.
+
+You may hear team members refer to "licking the cookie". Assigning a
+bug to yourself, or otherwise indicating that you will work on it,
+tells others on the team to not fix it. If you then don't work on it
+straight away, you are acting like someone who has taken a cookie,
+licked it to unappetising to other people, and then not eaten it right
+away. By extension, "unlicking the cookie" means indicating to the
```

Comment:
I've never heard of unlicking the cookie. :)  But it's not a bad idea to mention the idea that it's good to unassign bugs you aren't working on so others feel more empowered to take them?